index.html

```diff
@@ -31,19 +30,17 @@
         </div>
 
     </header>
-    <main class="main-section">
         <section class="head-part">
             <h1>I'm Joel <br>Glad to see<br> you!</h1>
             <p>I'm a software developer! I can help you build a product , feature or website Look through some of my
             work and experience! If you like what you see and have a project you need coded, don't hesitate to
             contact me.</p>
-
-                <span class="paragraph">LETS CONNECT</span>
-                <div class="social_icons">
-                    <ul class="font_icons">
-                        <li>
-                            <a href="#">
-                                <img alt="twitter" src="images/twitter-icon.png"/>
+            <span class="paragraph">LETS CONNECT</span>
+            <div class="social_icons">
```

Comment:
- [ ] Please use a `<hx>` tag for the "LETS CONNECT" call to action as it is much more semantically correct than simply using a `<span>`